Sources/TensorFlow/Core/TensorShape.swift

```diff
@@ -73,45 +73,98 @@ public struct TensorShape: ExpressibleByArrayLiteral {
     }
 }
 
-public extension TensorShape {
+extension TensorShape: Collection, MutableCollection {
```

Comment:
Please separate `extension TensorShape: Collection` and `extension TensorShape: MutableCollection`. Make each extension contain only protocol requirement implementations for that protocol, please.

Reply:
Never mind, resolved: https://github.com/tensorflow/swift-apis/pull/615#discussion_r365906757